backend/app/llm/chat.py

```
@@ -0,0 +1,22 @@
+from langchain_google_genai import ChatGoogleGenerativeAI
+from langchain_core.messages import HumanMessage
+from app.core.config import settings
+
+async def chat_completion(prompt: str, context: dict | None = None) -> str:
```

Ruff (0.14.14)

[warning] Unused function argument: `context`

(ARG001)